.eslintcache

```diff
@@ -0,0 +1 @@
20","hashOfConfig":"13"},{"size":3337,"mtime":1639581772947,"results":"21","hashOfConfig":"13"},{"size":724,"mtime":1639581772948,"results":"22","hashOfConfig":"13"},{"size":654,"mtime":1642001565857,"results":"23","hashOfConfig":"13"},{"filePath":"24","messages":"25","errorCount":0,"fatalErrorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0,"usedDeprecatedRules":"26"},"bbo8g9",{"filePath":"27","messages":"28","errorCount":0,"fatalErrorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},{"filePath":"29","messages":"30","errorCount":0,"fatalErrorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0,"usedDeprecatedRules":"26"},{"filePath":"31","messages":"32","errorCount":0,"fatalErrorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0,"usedDeprecatedRules":"26"},{"filePath":"33","messages":"34","errorCount":0,"fatalErrorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0,"usedDeprecatedRules":"26"},{"filePath":"35","messages":"36","errorCount":0,"fatalErrorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0,"usedDeprecatedRules":"26"},{"filePath":"37","messages":"38","errorCount":0,"fatalErrorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0,"usedDeprecatedRules":"26"},{"filePath":"39","messages":"40","errorCount":0,"fatalErrorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0,"usedDeprecatedRules":"26"},{"filePath":"41","messages":"42","errorCount":0,"fatalErrorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0,"usedDeprecatedRules":"26"},{"filePath":"43","messages":"44","errorCount":0,"fatalErrorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0,"usedDeprecatedRules":"26"},{"filePath":"45","messages":"46","errorCount":0,"fatalErrorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0,"usedDeprecatedRules":"26"},"/Users/alexasche/Desktop/repos/contentful-app/src/index.tsx",[],["47","48"],"/Users/alexasche/Desktop/repos/contentful-app/src/components/ConfigScreen.tsx",[],"/Users/alexasche/Desktop/repos/contentful-app/src/components/Dialog.tsx",[],"/Users/alexasche/Desktop/repos/contentful-app/src/components/Page.tsx",[],"/Users/alexasche/Desktop/repos/contentful-app/src/components/Field.tsx",[],"/Users/alexasche/Desktop/repos/contentful-app/src/components/EntryEditor.tsx",[],"/Users/alexasche/Desktop/repos/contentful-app/src/components/Sidebar.tsx",[],"/Users/alexasche/Desktop/repos/contentful-app/src/components/Paginator.tsx",[],"/Users/alexasche/Desktop/repos/contentful-app/src/components/ResourceListItem.tsx",[],"/Users/alexasche/Desktop/repos/contentful-app/src/queries/hail-frequency.tsx",[],"/Users/alexasche/Desktop/repos/contentful-app/src/queries/warp2.tsx",[],{"ruleId":"49","replacedBy":"50"},{"ruleId":"51","replacedBy":"52"},"no-native-reassign",["53"],"no-negated-in-lhs",["54"],"no-global-assign","no-unsafe-negation"]
```

Comment:
I think we need to remove this file from version control and add `.eslintcache` to `.gitignore`.